src/components/MusicSheet.vue

```diff
@@ -142,7 +143,7 @@ export default {
     compositions: {
       patched (data) {
         // Called whenever a composition belonging to this user has been updated
-        console.log("recieved this", this.username, data.active)
+        console.log('recieved this', this.username, data.active)
```

Comment:
apparently i can't leave the comment where i wanted to but for line 134, can you change msg-> cur_username